scss/_patterns_navigation.scss

```diff
@@ -876,7 +873,7 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     }
     .p-navigation__nav {
       display: block;
-      height: 100vh;
+      height: calc(100vh - $navigation-height);
```

Comment:
Good call, but indeed, it was using 100vh before, so I just adjusted it based in height of the nav.

And I'd really want to avoid using JS to handle this.

But, I guess it's something to keep in mind and test, if it turns out buggy or problematic.

Reply:
Unfortunately, I don't think we have a choice but to handle it in JS. I tried it on my phone and there is definitely lost content.
Android chrome:
![0](https://github.com/user-attachments/assets/4e9c7653-2647-4322-925f-d52282c0157c)
Linux chrome:
![image](https://github.com/user-attachments/assets/2500d7e2-ef89-469c-82ec-9b8aa5ea202d)

I don't using `vh` for the initial setting is the problem here, especially as it will be reset in for these specific cases.

Reply:
Partially it's the bottom example toolbar that is covering the bottom. So it shouldn't be a problem on desktop.

But seems to be an issue on mobile indeed.

Maybe adding some arbitrary margin/padding on bottom would remedy the issue at least partially.

Reply:
Bottom margin/padding seems to have helped. Here's a view of Firefox simulating Android with the example toolbar removed using inspector:

<img width="385" alt="Screenshot 2024-08-08 at 12 17 12 PM" src="https://github.com/user-attachments/assets/c1d25b53-c172-494e-a4b3-84033fdf0262">

#5283 was created to prevent this sort of thing from blocking view on examples page in future

Reply:
For whatever reason I don't understand height 100% doesn't seem to work here. Height of parent element is limited to screen size and it scrolls off screen.

Now that I write it I feel like maybe this also is related to the 100vh being used.

Anyway, I'd prefer to tackle it separately, now that it somehow works fine here. But I agree, that current implementation is not ideal and should be improved.

Reply:
To be revisited in https://warthogs.atlassian.net/browse/WD-14056